src/sysinfo/nvapi_adapter.h

```diff
@@ -48,17 +58,19 @@ namespace dxvk {
       private:
         Vulkan& m_vulkan;
         Nvml& m_nvml;
+        Com<IDXGIAdapter3> m_dxgiAdapter;
```

Comment:
I guess no way around to keep an adapter alive. I hope this won't give us tear down issues in titles.

Reply:
Unless adapters are singletons (which I don't think they are) it should be fine. But if it proves to be a problem we can always try switching to manual usage of `VK_EXT_memory_budget` on our own.